umbraco/Umbraco-CMS
# Skip move logic if the parent is staying the same

### Prerequisites


### Description
Added checks to ContentTypes, Content and DataTypes Move functions to skip all the logic if the parent would stay the same.

### How to test
Try moving around ContentTypes, Content and DataTypes and make sure this still works as expected
